ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -400,34 +393,16 @@ def mark_datamodels_as_deleted(self) -> Iterable[Either[DeleteEntity]]:
                 params={"service": self.context.dashboard_service},
             )
 
-    def process_owner(self, dashboard_details):
+    def process_owner(  # pylint: disable=unused-argument, useless-return
```

Comment:
nit - Not sure that I like the `process_owner` name. Maybe a `get_owner_ref` might help us understand what is actually going on

Reply:
Changed the name